nomos-libp2p/src/behaviour/nat/swarm_ext.rs

```diff
@@ -0,0 +1,11 @@
+use libp2p::Multiaddr;
+
+use crate::Swarm;
+
+impl Swarm {
+    pub fn add_external_address_candidate(&mut self, addr: Multiaddr) {
```

Comment:
Didn't we say this happens automatically when listening by `Identify`? Why do we need to expose another method? We want to force clients to start listening on a given address, no? I think we can add this method later on when we start dealing with circuit-based NAT traversal. Until then, I don't think we need this, do we?